_posts/2024-10-24-test-markdown.md

```
@@ -0,0 +1,122 @@
+---  
+
+参考链接：
+```test
+https://before80.github.io/go_docs/goBlog/2023/ForwardCompatibilityAndToolchainManagementInGo1_21/
+```
+
+# Go 工具链管理（Toolchain Management）总结
+
+从 Go 1.21 开始，go 命令不仅可以自动下载所需的 Go 模块，还可以自动下载和更新 Go 工具链，这类似于 Node 的 nvm 和 Rust 的 rustup，但这是内置在 go 命令中的功能，而不是一个单独的工具。
+
+### 自动下载和更新工具链：
```

Markdownlint

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)